src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -210,4 +212,10 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
 
     return error.error;
   }
+
+  private displayFileWaitingStatus () {
+
+    this.valid = false;
+    this.uploadError = WriteDocumentFieldComponent.UPLOAD_WAITING_FILE_STATUS;
```

Comment:
are we using the `uploadError` to display generic info messages as well? It's not his purpose. Perhaps we could improve this by either renaming `uploadError` to something more generic, or introducing a new property for displaying info messages so we clearly separate the 2